std_msgs/Header header
diff_drive/Constants constants
diff_drive/Linear linear
diff_drive/Angular angular
bool forward_movement_only
bool end_of_path_stop
float64 number_of_paths
float64 path_index
diff_drive/Goal[] goals